webpack.prod.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const { getBaseConfig } = require('@edx/frontend-build');
+
+const origConfig = getBaseConfig('webpack-prod');
+
+const config = {
+  ...origConfig,
```

Comment:
I use the orig config from base frontend-build in case enhancements are needed, but I still largely use the rest of the config as it was before, I did not research into whether I could skip the optimizations override, I just wanted to get one file for now in the output but we could research into using the chunks for index.js later